### ISSUE TITLE
[feature request] compatibility with Tab Center

### ISSUE THREAD

**Issue by andreykurilin** [piroor/treestyletab#1231]
Hi!
Currently, FireFox team works on several experimental features (testpilot). In near future some of them(features) will move to main pipeline. One of this feature is [TabCenter](https://testpilot.firefox.com/experiments/tab-center/) - a convenient(more convenient than default one) displaying of tabs.

TabCenter team doesn't plan to implement tab inheritance by itself:

> One of the explicit goals of this add-on is to show a simpler interface than Tree Style Tabs, since most people don't really understand (or don't care about) hierarchies in their tabs, so we're not going to implement this feature. 

What do you think about integration with TabCenter addon ?

**Comment by piroor** [comment]
Oops, I forgot to respond. See also these similar topics:

 * https://github.com/piroor/treestyletab/issues/1255#issuecomment-285235279 (my recent response for a dup of this)
 * https://github.com/piroor/treestyletab/issues/1224

**Comment by andreykurilin** [comment]
@piroor : I should say sorry. When I found a native addon for FireFox and I saw screenshots of it, I thought it could be a good idea to integrate your addon to native one, or at least make them compatible. Even more, I wrote them about an implementation of tree-style (the answer was that they do not want to be multipurposed plugin, they want to over just one case..). It was a first look at the issue.
After that, I tried to use TC for some time, and I found that it is not convenient at all for me. Your plugin is much better and I do not think that any kind of integration should exist :) at least I'm totally satisfied with your work. Thanks!
